tools/ovc/openvino/__init__.py

```diff
@@ -43,12 +43,13 @@
 from openvino.runtime import Tensor
 from openvino.runtime import OVAny
 
-from openvino.runtime import compile_model
+# Helper functions for openvino module
+from openvino.runtime.utils.data_helpers import tensor_from_file
+from openvino._ov_api import compile_model
```

Comment:
answered above

Reply:
It is not clear. Let us use `from openvino import compile_model`. Now it should work.

Reply:
`_ov_api.py` is just a place for Python wrappers for C++ bindings (sometimes they're necessary, not in all cases). Since all `__init__.py` files are merged into one in wheel, they have to be aligned - that's why that change is also in `tools/ovc/openvino/__init__.py`.

Because it's in `__init__.py`, this import is indeed "internal" as you mentioned - the user will still be able to run `from openvino import compile_model`, because that `__init__.py` pulls `compile_model` from `_ie_api.py` and publishes it in the main `openvino` module.

The act of publishing the function to the main module itself is internal, it's not public API and should not concern the user, who keeps the same functionality of `from openvino import compile_model`.